psycopg/python.h

```diff
@@ -34,8 +34,8 @@
 #error "psycopg requires Python 2.7"
 #endif
 
-#if PY_3 && PY_VERSION_HEX < 0x03050000
-#error "psycopg requires Python 3.5"
+#if PY_3 && PY_VERSION_HEX < 0x03060000
```

Comment:
Unrelated, but in this file there is an ifdef to Python 3.2: could you please drop it?

Reply:
This bit? Isn't it used by Python 2.7?


```h
/* hash() return size changed around version 3.2a4 on 64bit platforms.  Before
 *   this, the return size was always a long, regardless of arch.  ~3.2
 *   introduced the Py_hash_t & Py_uhash_t typedefs with the resulting sizes
 *   based upon arch. */
#if PY_VERSION_HEX < 0x030200A4
typedef long Py_hash_t;
typedef unsigned long Py_uhash_t;
#endif
```

Reply:
Ah right, I guess it is. I thought it was into an "if PY3" branch.